modules/extensions/process/publisher/process/themes/default/partials/add_doc.hbs

```diff
@@ -0,0 +1,136 @@
+                                                    name="docUrl">
+                                            </div>
+                                            <label class="radio" style="margin-left:20px">
+                                                <input type="radio" name="optionsRadios1"
+                                                       id="optionsRadios8"
+                                                       value="file"
+                                                       onclick="javascript:$('#sourceFile').toggle('slow');$('#sourceUrlDoc').hide('slow')">Document
+                                                <a class="icon-question-sign help_popup"
+                                                   data-original-title=""
+                                                   title=""></a>
+
+                                                <p id="fileDoc_help" class="hide">Upload a
+                                                    documentation file</p>
+                                            </label>
+
+                                            <div id="fileNameDiv" style="display:none;"></div>
+                                            <a onclick="javascript:$('#sourceFile').toggle('slow')"
+                                               id="toggleFileDoc"
+                                               style="display:none;">
+                                                Change File
+                                            </a>
+
+                                            <div id="sourceFile" class="controls"
+                                                 style="display:none;">
+                                                                        <span class="requiredAstrix"
+                                                                              style="color: red;">*</span> <input
+                                                    type="file" class="required"
+                                                    style="width:98%"
+                                                    name="docLocation"
+                                                    id="docLocation">
+
+                                                <p id="fileType_help">eg : .docx, .doc</p>
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                            <div style="clear:both; font-size:1px;"></div>
+                        </div>
+                        <br/>
+                        <div class="row-fluid">
+                            <div class="span12">
+                                <div class="control-group">
+                                    <!--<button type="button" class="btn btn-primary" id="saveDocBtn" onclick="saveDoc()">Add Document</button>-->
```

Comment:
Fixed in https://github.com/wso2/product-pc/commit/e931bb704e9065cf57f08f61619a45a4beecb5c7